### ISSUE TITLE
getProfile returns incorrect followingCount

### ISSUE THREAD

**Issue by ThijnK** [the-convocation/twitter-scraper#66]
In particular, it seems that the `followingCount` property of the `Profile` object is copying the `likesCount` property.

For example, this is (part of) one of the profiles I fetched:
```
{
  ...
  biography: '',
  followersCount: 3,
  followingCount: 82,
  friendsCount: 5,
  mediaCount: 0,
  isPrivate: false,
  isVerified: false,
  likesCount: 82,
  listedCount: 0,
  ....
}
```
I checked and the `followersCount` here is incorrect, but equal to the `likesCount`.
Tested with some other profiles and the patterns repeats itself

I'm on the most recent version

**Comment by ThijnK** [comment]
It seems that Twitter itself actually just uses the `friends_count` as the following count, and that is already being returned by the `getProfile`. 
So as far as I know, you can just use the `friendsCount` entry as the nr of following.
Still, may as well remove the `followingCount` property, or give it the same value as `friendsCount`.

**Comment by karashiiro** [comment]
Fixed in #67
